app/javascript/containers/App.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Router, Route, IndexRoute, browserHistory } from 'react-router';
+import HomePage from './HomePage'
+import MostRecentSightings from '../components/MostRecentSightings'
+
+const App = props => {
+
+  return(
+    <Router history={browserHistory}>
+      <Route path='/' >
```

Comment:
Since you don't have a component that's showing on all pages (i.e. a NavBar), we actually don't want to nest this. I'd remove lines 10 and 13, and change line 11 to be:
```
<Route path='/' component={HomePage} />
```